scripts/imu_spawners/spawn_imu.py

```diff
@@ -114,23 +115,32 @@ def set_pose(self, name, pose):
         `poses`: `geometry_msgs.msg.Pose`
             The pose of the IMU
         """
+        init_pose = Pose()
```

Comment:
why did you copy `pose` to `init_pose`?

Reply:
I had to do this because pose.position.x doesn't exist, only pose.position[0...2].